### DISCUSSION TITLE
Welcome to SharkDevelop3 Discussions!

### DISCUSSION THREAD

**Discussion by LucasSGallo** [CodeSystem2022/SharkDevelop3#101] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LucasSGallo** [comment]
Chicos! Son unos genios! Le pusieron nombre de fecha en vez de leccion! Capos

**Comment by meliriera** [comment]
![bitmoji](https://sdk.bitmoji.com/render/panel/8b261cbb-ac37-400f-8d21-1e5e3be47d2f-95c306cf-7bf5-4041-8689-e525dbe362ea-v1.png?transparent=1&palette=1&width=246)
